variables.tf

```diff
@@ -4,36 +4,85 @@
 # You must provide a value for each of these parameters.
 # ------------------------------------------------------------------------------
 
-variable "subnet_id" {
-  type        = string
-  description = "The ID of the AWS subnet to deploy into (e.g. subnet-0123456789abcdef0)"
+variable "auditors" {
+  # Currently-defined audit roles: financial_audit, security_audit
+  description = "A map containing the usernames of each auditor and a list of audit roles assigned to that user.  Example: { \"firstname1.lastname1\" = { \"roles\" = [ \"financial_audit\", \"security_audit\" ] },  \"firstname2.lastname2\" = { \"roles\" = [ \"security_audit\" ] } }"
+  type        = map
 }
 
 # ------------------------------------------------------------------------------
 # OPTIONAL PARAMETERS
 #
 # These parameters have reasonable defaults.
 # ------------------------------------------------------------------------------
-variable "ami_owner_account_id" {
-  type        = string
-  description = "The ID of the AWS account that owns the Example AMI, or \"self\" if the AMI is owned by the same account as the provisioner."
-  default     = "self"
+
+variable "assume_master_financialaudit_policy_description" {
```

Comment:
These variables should all have `type`s.

Reply:
Thanks- I copied and pasted these from `cool-users-non-admin` and we didn't have them there, so good catch!  Fixed in https://github.com/cisagov/cool-auditor-iam/pull/1/commits/6349a7bdfba86eeacc49c595e146f61c26a68db1.